app/components/cms_hero_component.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CmsHeroComponent < ViewComponent::Base
+  include ViewComponent::Translatable
+
+  def initialize(resource, title:)
+    @resource = resource
+    @title = title
+  end
+
+  def call
+    render HeroComponent.new(title: @title)
```

Comment:
I was going to leave this job for the new junior developer as I think it will be a good starting task to go through and understand the code base a bit better and get familiar with the layouts